playwright.config.ts

```diff
@@ -42,6 +46,9 @@ const config: PlaywrightTestConfig = {
       // test name. We omit the project name here to keep snapshot names tidy.
       name: '',
       grep: /@agnostic/,
+      // put css snapshots in test filename subdirectories
+      snapshotPathTemplate:
+        '{testFileDir}/{testFileName}-snapshots/{arg}{-projectName}{ext}',
```

Comment:
```suggestion
        '{testFileDir}/{testFileName}-snapshots/{arg}{-projectName}{ext}',
```
```suggestion
        '{testFileDir}/{testFileName}-snapshots/{arg}-{projectName}{ext}',
```

Reply:
In fact, this project has an empty string as a name, so the `{projectName} template is redundant. I'd just remove it.

Reply:
ah, makes sense. Was wondering why this didn't break anything.